### DISCUSSION TITLE
设置倾斜角，左键移动到此处，FlyTo有位置有偏差

### DISCUSSION THREAD

**Discussion by mc15829702687** [zouyaoji/vue-cesium#297] [Q&A]
请教大佬，点击地图上某点获取位置后，调用 flyTo 方法，由于初始化时设置了方位角、倾斜角等，位置会有偏差，该怎么解决，查找了好多方法效果都不理想

**Comment by zouyaoji** [comment]
我一般是先调到我想要的视角，然后通过viewer.camera把相机参数的position、heading、pitch、roll都读出来，然后用这些参数飞。
你做个收藏视角的功能，就保存上述参数，每次飞到这些视角即可。

**Reply by mc15829702687** [reply]
大佬，一开始初始化的时候设置了这些视角，飞行前也能拿到，之后飞行的时候传入这些参数与位置信息，就有偏差了

**Reply by zouyaoji** [reply]
这就没遇到过了

**Reply by mc15829702687** [reply]
麻烦大佬了，这么忙还抽时间回答我问题，感谢
